src/utils/HttpError.ts

```diff
@@ -0,0 +1,7 @@
+export class HttpError extends Error {
```

Comment:
We can import this from express-utilities once published but for now this is good.

Reply:
indeed, I await patiently :)